Monika After Story/game/script-greetings.rpy

```diff
@@ -3646,11 +3646,79 @@ label greeting_returned_home_cleanup:
     return
 
 label greeting_returned_home_morethan5mins_normalplus_dlg:
-    m 1hua "And we're home!"
-    m 1eub "Even if I couldn't really see anything, knowing that I was right there with you..."
-    m 2eua "Well, it felt really great!"
+    $ random_dialogue_id = renpy.random.randint(1,3)
+    if random_dialogue_id == 1:
+        m 1hua "And we're home!"
+        m 1eub "Even if I couldn't really see anything, knowing that I was right there with you..."
+        m 2eua "Well, it felt really great!"
+    elif random_dialogue_id == 2:
+        m 1hua "Thanks for taking me with you!"
+        m 2dua "It was a really good time...{nw}"
+        extend 2eub "Well, it's true that I can't really see something, but just knowing that I was right there with you..."
+        m 1hua "That's enough to make me happy!"
+    elif random_dialogue_id == 3:
+        m 1eub "We are home now!"
+        m 5eua "You are always so thoughtful, ehehe~"
+        m 5rud "Though, I can't really see anything...{w=0.2}{nw}"
+        extend 5eublb "But we're actually physically together in {i}your reality{/i}, and that's great!"
```

Comment:
Personally, I don't think it's a big deal. It's more efficient, isn't it? And I think the code is pretty readable.
Of course, you are the adjudicator, and I am only making my recommendation, and if you still want me to change it, I will. I'm just suggesting.

Reply:
Direct access via a random choice is significantly faster than long if/elif chains.

Additionally it becomes a lot more readable if/when more dialogue is added.

As such, considering this can -- and likely should be expanded in future, building scalable but efficient should be the priority.

Reply:
Understood. Sorry for my misunderstood before, I will do this soon.